### PR TITLE
PT# 138279805 Flatten tab navigation structure

### PR DESCRIPTION
**Pull request by AllenBW**
https://www.pivotaltracker.com/story/show/138279805

@miq-bot add_label enhancement, euwe/no 

@ManageIQ/core-ui-service as per our discussions
screen shot 2017-01-24 at 4 09 18 pm

this ticket only encompasses the left nav 

TODO: 
- https://www.pivotaltracker.com/story/show/138278105
- https://www.pivotaltracker.com/story/show/138278145
- https://www.pivotaltracker.com/story/show/138278069

### CODE REVIEW

File: client/app/config/navigation.config.js [hunk 1]
```diff
@@ -2,103 +2,61 @@
...
-    'fa fa-copy',
-    N_('Total available catalog items')
+  const orders = createItem(
+    N_('Orders'),
+    'orders',
+    'fa fa-file-o',
+    N_('Total orders submitted')
+  );
+  const requests = createItem(
+    N_('Request'),
```
Comment (jjlangholtz) [inline client/app/config/navigation.config.js]: Should this line be `Requests` instead of `Request`?

Comment (AllenBW) [inline client/app/config/navigation.config.js]: YES

**Comment by serenamarie125** [comment]
Services & Orders should be My Services & My Orders as shown in the design doc found here: https://github.com/ManageIQ/manageiq-design/blob/master/UX/ui-service/Framework/design.md

**Comment by chriskacerguis** [comment]
@serenamarie125 Does the "My" make sense, when an Admin goes to that page, it would show them everything.  Thoughts?

**Comment by serenamarie125** [comment]
@chriskacerguis yes.  We have had this discussion a number of times, and I've talked about it with John Hardy as well who agrees that My should be included.

**Comment by chriskacerguis** [comment]
@AllenBW can you please update the conflict, and update the wording?

**Comment by AllenBW** [comment]
Cool, well ok then, "My" has been added
@chriskacerguis

**Comment by AllenBW** [comment]
@jeff-phillips-18 
yep! already a story out for that [here]( https://www.pivotaltracker.com/story/show/138278105)
[this one is for updating routing](https://www.pivotaltracker.com/story/show/138278145)
[and this one folder structure](https://www.pivotaltracker.com/story/show/138278069)

can ya think of anything else we might have missed?

**Comment by jeff-phillips-18** [comment]
Nothing I can think of

**Pull request merged**